README.md

```diff
@@ -121,15 +127,27 @@ This is a  folder with three files per document:
 * ``doc.txt`` : document content
 
 As above, you may omit the ``doc.tsv`` if you just want to
-predict the quality of the MLC result.
+predict the quality of the MLC result and you may omit the ``doc.annif`` if you provide
+the split for Label Calibration training.
 
 ### Train
 To train a model, use the ``train`` mode, e.g.:
 
 ``qualle train /path/to/train_data_file /path/to/output/model``
 
-It is also possible to use label calibration (comparison of predicted vs actual labels) using the subthesauri of a thesaurus (such as the [STW](http://zbw.eu/stw/version/latest/about))
-as categories (please read the paper for more explanations). Consult the help (see above) for the required options.
+Qualle uses a feature called Label Calibration, which is a multi-output regression model that provides expectations
+of the correct number of concepts for a document. This model is trained before the quality estimation regressor (Recall Predictor)
+is trained.
+You can provide a separate training data split for Label Calibration:
+
+``qualle train /path/to/train_data_file /path/to/output/model --label-calibration-train-data /path/to/lc_train_data_file``
+
+If a separate training data split is not provided, the Label calibrator will be trained with the same training data split as the
+Recall Predictor, and the required input for training the Recall Predictor is provided by
```

Comment:
Just curious: What could be negative effects of using the same training data set for both Label Calibration and Recall Prediction? Is it possible to add one sentence to make the decision easier or at least to make users aware of that? (I understand that on the other hand providing a separate split is only possible if you have enough data.)

Reply:
So the recall predictor needs input from the label calibrator. If a separate split is not provided, the label calibrator will be trained on the same split. But we can't use this trained calibrator to create the input for the predictor since it's trained on the same data and is therefore overfitted.
So we use cross-validation, which technically means
further label calibrators x_1,...,x_n are used to provide the input. So, in summary, the quality estimation is performed with a different label calibrator than the one used for the training.

In addition, the label calibrators x_1,...,x_n are trained on a part of the train data used by the recall predictor. This means that the recall predictor is not trained on "unseen" data, which can lead to bias.

If you provide a separate split, the overall quality estimate will be cleaner because you are using the same label calibrator in training and in prediction, and the recall predictor has been trained on "unseen data".

It's hard to say for sure whether one approach or the other leads to better results, but I want to give the users of Qualle a choice.

Reply:
So, I would suggest to append following text:

"This implies that the quality estimation is not performed with the exact same Label Calibrator used in the training, 
and since the trained calibrators from the cross-validation use parts of the same training data as the Recall Predictor,
the Recall Predictor is not trained on "unseen" data, what could lead to a bias.
Whether one or the other approach has a negative effect must be tested individually."

Reply:
Okay! I trust that only people who are able to make that call will use qualle anyway.